### DISCUSSION TITLE
Mancanza in home assistant installato su debian in docker  del menu componenti aggiuntivi?

### DISCUSSION THREAD

**Discussion by schdavide** [indomus/forum#2348] [HUB: Home Assistant]
Buongiorno, 
            sono un neofita di HA ho un mini PC MSI con un Celeron e 8Gb di ram e 256Gb di SSd, Ho installato seguendo un guida Ha in Docker e dopo averlo lanciato e funziona anche bene, ma non c'è il meno COMPONENTI AGGIUNTIVI, ed ho dovuto installare HACK per riuscire a trovare le integrazioni per sonoff e altri domotici che ho in casa, ma di altri che li non trovo, devo ottenerli o da componenti aggiuntivi, oppure editando il file "configuration.yaml", che non riesco a trovare e di cui non riesco a scaricare il programma file editor, che si dovrebbe trovare in componenti aggiuntivi, qui si innesca il loop.
Aiuto chi ne sa più di me mi dia una mano per favore.
Davide

**Comment by snakuzzo** [comment]
Ti sei perso un pezzo...

I componenti aggiuntivi (add-on) sono presenti solo nelle istallazioni HA OS e HA Supervised.
Su core nudo e crudo (che sia su docker o meno) non esistono. Tutto quello che fanno gli addon te lo devi fare tu a "manella" a parte

**Reply by schdavide** [reply]
OK grazie dell info, quindi mi consigli di reinstallare una versione HA OS o una HA Supervised? e se si, con il mio HD mi consiglieresti uan guida, per installare un HA OS o un HA Supervised, sappi che il mini pc non mi serve per altro oltre HA?

**Reply by snakuzzo** [reply]
Le guide le trovi sul sito.
Io non ti consiglio niente, perché la scelta dell'installazione deve essere tua, anche e soprattutto in base alle tue capacità

Leggi -> https://indomus.it/focus/home-assistant-hassio-o-home-assistant-core-quale-installazione-fa-al-caso-mio/
